memcached-image/config.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+# that's what gets extracted from the archive (see the .kiwi file)
```

Comment:
Where is the .kiwi image? (sorry for my ignorance)

Reply:
In the OBS/IBS package. We won't store it in git because the .kiwi files are autogenerated.

Reply:
Do you have a link to an example package/project?

Reply:
Yes. See description (https://build.opensuse.org/package/show/home:tbechtold:branches:openSUSE:Templates:Images:Tumbleweed/memcached-image) . But the _service file needs adjustments to fetch the sources from git.